src/modules/analysis/__tests__/analysisAddComponent-test.js

```diff
@@ -55,17 +59,33 @@ const form = fieldsArray.reduce(
   {}
 );
 
+const appSession = {
+  museumId: 99,
+  accessToken: '1234'
+};
+
 describe('AnalysisAddComponent', () => {
   it('saveAnalysisEventLocal should call saveAnalysisEvent', () => {
     const saveAnalysisEvent = sinon.spy();
-    const appSession = {
-      museumId: 99,
-      accessToken: '1234'
-    };
-    saveAnalysisEventLocal(appSession, form, store, saveAnalysisEvent)();
-    expect(saveAnalysisEvent.callCount).toBe(1);
-    expect(saveAnalysisEvent.getCall(0).args[0].museumId).toEqual(99);
-    expect(saveAnalysisEvent.getCall(0).args[0].token).toEqual('1234');
+    saveAnalysisEventLocal(appSession, form, store, saveAnalysisEvent)(1);
+    expect(saveAnalysisEvent.calledOnce).toBe(true);
+    expect(saveAnalysisEvent.getCall(0).args[0].museumId).toBe(99);
+    expect(saveAnalysisEvent.getCall(0).args[0].token).toBe('1234');
+    expect(saveAnalysisEvent.getCall(0).args[0].data.analysisTypeId).toBe(
+      'b15ee459-38c9-414f-8b54-7c6439b44d3d'
+    );
+  });
+
+  it('meh', done => {
+    const fakeGoTo = sinon.spy();
+    const fakeFn = () => Observable.of(null).toPromise();
+    const fn = goToAnalysis(fakeFn, appSession, fakeGoTo);
+    const analysisId = 2;
+    fn(analysisId).then(() => {
+      expect(fakeGoTo.calledOnce).toBe(true);
+      expect(fakeGoTo).toMatchSnapshot();
```

Comment:
Got below error on it.
Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.